pinot-plugins/pinot-input-format/pinot-csv/pom.xml

```diff
@@ -43,5 +43,11 @@
       <groupId>commons-lang</groupId>
       <artifactId>commons-lang</artifactId>
     </dependency>
+    <dependency>
```

Comment:
is this needed?

Reply:
The build failed with a dependency error without it.